jwst/tests_nightly/general/fgs/test_guider_pipeline.py

```diff
@@ -0,0 +1,89 @@
+import os
+import pytest
+from astropy.io import fits as pf
+from jwst.pipeline.calwebb_guider import GuiderPipeline
+
+
+pytestmark = [
+    pytest.mark.usefixtures('_jail'),
+    pytest.mark.skipif(not pytest.config.getoption('bigdata'),
+                       reason='requires --bigdata')
+]
+
+
+def test_guider_pipeline1(_bigdata):
+    """
+
+    Regression test of calwebb_guider pipeline performed on ID-image data.
+
+    """
+
+    GuiderPipeline.call(_bigdata+'/fgs/test_guiderpipeline/jw88600073001_gs-id_7_image-uncal.fits',
+                        output_file='jw88600073001_gs-id_7_image-cal.fits')
+
+    # Compare calibrated ramp product
+    n_cr = 'jw88600073001_gs-id_7_image-cal.fits'
+    h = pf.open( n_cr )
```

Comment:
Nitpick: Would prefer `with pf.open(n_cr) as h` context manager. But this can probably be addressed in a future sprint or something.

In addition, the `fitsdiff` logic seems repetitive across the board. Perhaps that whole logic block can be turned into an utility function in the future to minimize code duplication.

Reply:
Agreed. I originally intended to replace these redundant `fitsdiff` calls with a generic wrapper, but I was told not to change the tests (too much) so I left them alone.

(This will probably need its own issue)

Reply:
Nitpick+1: Is there a good reason to continue importing this as `pf` (pyfits)?
Why not `from astropy.io import fits` then `with fits.open(...) as h`?